.gitmessage

```diff
@@ -0,0 +1,5 @@
+<type>[optional scope]: <description>
+
+[optional body]
+
+Refs: ID-123
```

Comment:
The extra keystrokes to delete these prompts might become annoying, the placeholders could be mentioned in comments instead.

Also, I think the way I've seen the optional scope used, it has another set of parentheses - e.g. `chore(tests): add test for...`